### PR TITLE
docs: update a final Go Cloud reference

### LINKED ISSUE #1186: all: "Go Cloud" to "Go Cloud Development Kit"

### PR DESCRIPTION
**Pull request by vangent**
Fixes #1186.

### CODE REVIEW

**Comment by vangent** [comment]
ping @ijt

**Comment by ijt** [comment]
Looking. Sorry for the delay.

File: doc.go [hunk 1]
```diff
@@ -15,21 +15,21 @@
...
+It does this by providing stable, idiomatic interfaces for common uses like
+storage and databases. Think `database/sql` for cloud products.
 
-At the core of the project are common types implemented by cloud providers.
-For example, the blob.Bucket type can be created using
-gcsblob.OpenBucket, s3blob.OpenBucket, or any other provider. Then,
-the blob.Bucket can be used throughout your application without worrying
-about the underlying implementation.
+At the core of the Go CDK are common types implemented by cloud providers.
+For example, the *blob.Bucket type can be created using gcsblob.OpenBucket,
```
Comment (ijt) [inline doc.go]: How about "objects of type blob.Bucket" instead of "the *blob.Bucket type"?

File: doc.go [hunk 2]
```diff
@@ -15,21 +15,21 @@
...
+It does this by providing stable, idiomatic interfaces for common uses like
+storage and databases. Think `database/sql` for cloud products.
 
-At the core of the project are common types implemented by cloud providers.
-For example, the blob.Bucket type can be created using
-gcsblob.OpenBucket, s3blob.OpenBucket, or any other provider. Then,
-the blob.Bucket can be used throughout your application without worrying
-about the underlying implementation.
+At the core of the Go CDK are common types implemented by cloud providers.
+For example, object of the *blob.Bucket type can be created using
```
Comment (ijt) [inline doc.go]: s/object/objects/

Comment (vangent) [reply doc.go]: Done.

**Pull request merged**